metagraph/types.py

```diff
@@ -125,4 +125,11 @@ class BipartiteGraph(AbstractType):
     unambiguous_subcomponents = {EdgeSet}
 
 
+#################################
+# Embedding
+#################################
+class NodeEmbedding(AbstractType):
+    pass
```

Comment:
This needs properties defined. The only one I can think applies is "dtype" of the matrix.
"is_square" doesn't apply -- we don't really care one way or the other.
"is_dense" must always be True
The node dtype is always NodeID.

Reply:
https://github.com/metagraph-dev/metagraph/pull/133/commits/eb0eda1c191b9170b685a88b55e1b3319b42d464 adds the abstract property `matrix_dtype` to `NodeEmbedding`.